cli/test/e2e/package_repository_test.go

```diff
@@ -64,6 +64,36 @@ func TestPackageRepository(t *testing.T) {
 		kubectl.Run([]string{"get", "pkg/pkg.test.carvel.dev.2.0.0"})
 	})
 
+	logger.Section("adding of existing repository", func() {
+		kappCtrl.Run([]string{"package", "repository", "add", "-r", pkgrName, "--url", pkgrURL})
```

Comment:
Let's also add some checks here. I think this would have succeeded before as well but it would have taken a lot of time.

Reply:
Additional checks added.